docs/src/examples/components/TextArea/Usage/TextAreaMaxLengthExample.shorthand.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react'
 import { TextArea } from '@stardust-ui/react'
 
 const TextAreaMaxLengthExample = () => (
```

Comment:
```suggestion
const TextAreaExampleMaxLength = () => (
```

Oops, was missed previously